mongo/metadata.csv

```diff
@@ -90,6 +90,20 @@ mongodb.mem.mapped,gauge,,mebibyte,,Amount of mapped memory by the database.,0,m
 mongodb.mem.mappedwithjournal,gauge,,mebibyte,,"The amount of mapped memory, including the memory used for journaling.",0,mongodb,memory mapped with journal
 mongodb.mem.resident,gauge,,mebibyte,,Amount of memory currently used by the database process.,0,mongodb,mem resident
 mongodb.mem.virtual,gauge,,mebibyte,,Amount of virtual memory used by the mongod process.,0,mongodb,mem virtual
+mongodb.metrics.commands.count.failed,rate,,commands,,Number of times "count" failed,-1,mongodb,command count failed
+mongodb.metrics.commands.count.total,gauge,,commands,,Number of times "count" executed,0,mongodb,command count executed
+mongodb.metrics.commands.createIndexes.failed,rate,,commands,,Number of times "createIndexes" failed,-1,mongodb,command createIndexes failed
```

Comment:
for the short form (last field of the CSV row, can we have `cmd` instead of `command`?